lib/genquery1/include/irods/private/genquery1_driver.hpp

```diff
@@ -0,0 +1,42 @@
+#ifndef IRODS_GENQUERY1_DRIVER_HPP
+#define IRODS_GENQUERY1_DRIVER_HPP
+
+struct GenQueryInp;
+
+#include "irods/private/genquery1_scanner.hpp"
+
+#include "parser.hpp" // Generated by Bison.
+
+#ifndef yyFlexLexerOnce
+#  undef yyFlexLexer
+#  define yyFlexLexer gq1FlexLexer
+#  include <FlexLexer.h>
+#endif // yyFlexLexerOnce
```

Comment:
Is this necessary?

Reply:
Yes because the lexer uses `%option prefix="gq1"`.

The use of `yyFlexLexerOnce` may be unnecessary though. Will investigate that.